lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java

```diff
@@ -112,6 +118,10 @@ public FusionEngine(Context context, Callback callback) {
     if (passiveInterval < Long.MAX_VALUE) {
       enablePassive(passiveInterval);
     }
+
+    if (prepLooper) {
+      Looper.loop();
```

Comment:
Moving the request to the main thread would not solve #131 entirely since as @amayatsky pointed out location updates would still stop if the app is killed by the system.

The root of the problem with this is that the framework's [FusionEngine](https://github.com/android/platform_frameworks_base/blob/master/packages/FusedLocation/src/com/android/location/fused/FusionEngine.java) runs inside system service and never shuts down and so updates via `PendingIntent` can be registered to wake up a service even if the app has been killed by the system.

The Lost [FusionEngine](https://github.com/mapzen/lost/blob/master/lost/src/main/java/com/mapzen/android/lost/internal/FusionEngine.java) on the other hand runs in a local service and so is tied to the application lifecycle.

Also we should do an explicit test of play services to see what happens if you call `requestLocationUpdates(...)` from a thread that has not called `Looper.prepare()`. Also I am wondering what play services does if updates are requested via `PendingIntent` and the app is killed by the user (rather than the system).

Really there are two issues here:
1. Should it be possible to request location updates from a thread that has not called `Looper.prepare()`?
2. Is there a way to ensure location updates via `PendingIntent` continue to fire even after the application has been killed by the system.